src/templates/show-view-edit.html

```diff
@@ -29,7 +29,7 @@ <h5 class='modal-title' id='importModalLabel'>{{ 'Import from file'|trans }}</h5
         </button>
       </div>
       <div class='modal-body'>
-        <p>{{ 'You can import data from a .eln archive, a .zip archive (created by eLabFTW) or a .csv file. %sSee documentation %s%s.'|trans|format("<a href='https://doc.elabftw.net/user-guide.html#importing-data' target='_blank'>", "<i class='fas fa-arrow-up-right-from-square'></i>", '</a>')|raw }}</p>
+        <p>{{ 'You can import data from a .eln archive, a .zip archive (created by eLabFTW) or a .csv file. When you upload a .csv file of items, additional columns are imported as metadata fields if they are present in the item type metadata. %sSee documentation %s%s.'|trans|format("<a href='https://doc.elabftw.net/user-guide.html#importing-data' target='_blank'>", "<i class='fas fa-arrow-up-right-from-square'></i>", '</a>')|raw }}</p>
```

Comment:
This should be removed (also to avoid breaking existing translations). And a complete documentation on the logic/features must be added in the documentation that is linked.